SECURITY.md

```diff
@@ -0,0 +1,30 @@
+<!-- AUTO-GENERATED, DO NOT EDIT! -->
+<!-- Please edit the original at https://github.com/ory/meta/blob/master/templates/repository/common/SECURITY.md -->
```

Comment:
Are you sure about that?

Reply:
This has to be removed, I have updated the file